Lib/gzip.py

```diff
@@ -23,7 +23,7 @@
 
 
 def open(filename, mode="rb", compresslevel=_COMPRESS_LEVEL_BEST,
-         encoding=None, errors=None, newline=None):
+         encoding=None, errors=None, newline=None, mtime=None):
```

Comment:
It is better to make it keyword-only parameter.
```suggestion
         encoding=None, errors=None, newline=None, *, mtime=None):
```

Also, it needs updating the module documentation and adding a What's New entry.

Reply:
Done. Thank you!